samples/snippets/src/main/java/com/examples/documentai/ParseForm.java

```diff
@@ -0,0 +1,101 @@
+ * limitations under the License.
+ */
+
+package com.examples.documentai;
+
+// [START document_quickstart]
+
+import com.google.cloud.documentai.v1beta2.Document;
+import com.google.cloud.documentai.v1beta2.DocumentUnderstandingServiceClient;
+import com.google.cloud.documentai.v1beta2.FormExtractionParams;
+import com.google.cloud.documentai.v1beta2.GcsSource;
+import com.google.cloud.documentai.v1beta2.InputConfig;
+import com.google.cloud.documentai.v1beta2.KeyValuePairHint;
+import com.google.cloud.documentai.v1beta2.ProcessDocumentRequest;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+public class ParseForm {
+  public static void parseForm() throws IOException, ExecutionException, InterruptedException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String location = "your-region";    // available regions https://cloud.google.com/compute/docs/regions-zones
+    String inputGcsUri = "gs://your-gcs-bucket/path/to/input/file.json";
+    parseForm(projectId, location, inputGcsUri);
+  }
+
+  public static void parseForm(String projectId, String location, String inputGcsUri)
+      throws IOException, ExecutionException, InterruptedException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (DocumentUnderstandingServiceClient client = DocumentUnderstandingServiceClient.create()) {
+      // Configure the request for processing the PDF
+      String parent = String.format("projects/%s/locations/%s", projectId, location);
+
+      KeyValuePairHint keyValuePairHint =
+          KeyValuePairHint.newBuilder().setKey("Phone").addValueTypes("PHONE_NUMBER").build();
+      KeyValuePairHint keyValuePairHint2 =
+          KeyValuePairHint.newBuilder()
+              .setKey("Contact")
+              .addValueTypes("EMAIL")
+              .addValueTypes("NAME")
+              .build();
+
+      FormExtractionParams params =
+          FormExtractionParams.newBuilder()
+              .setEnabled(true)
+              .addKeyValuePairHints(keyValuePairHint)
+              .addKeyValuePairHints(keyValuePairHint2)
+              .build();
+
+      GcsSource uri = GcsSource.newBuilder().setUri(inputGcsUri).build();
+
+      InputConfig config =
+          InputConfig.newBuilder().setGcsSource(uri).setMimeType("application/pdf").build();
+
+      ProcessDocumentRequest req =
+          ProcessDocumentRequest.newBuilder()
+              .setParent(parent)
+              .setFormExtractionParams(params)
+              .setInputConfig(config)
+              .build();
+
+      // Recognizes text entities in the PDF document
+      Document res = client.processDocument(req);
```

Comment:
Prefer full names, avoid abbreviations 
```suggestion
      Document response = client.processDocument(request);
```